src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -173,6 +173,44 @@ public class RestartComputerCommand : PSCmdlet, IDisposable
         private const string DefaultParameterSet = "DefaultSet";
         private const int forcedReboot = 6; // see https://msdn.microsoft.com/library/aa394058(v=vs.85).aspx
 
```

Comment:
Please update the link. (Now it is redirected).
And add doc link for new constants you are added below.